docs/UserGuide.md

```diff
@@ -9,331 +9,424 @@ Interface** (CLI) while still providing an easy way to visualize all events thro
+
+- Currently, we only support same-day events.
   Events which stretches across multiple days are not supported.
-* `undo` can only undo the most recent valid command.
-* Data files are not allowed to be modified directly. 
+- `undo` can only undo the most recent valid command.
+- Data files are not allowed to be modified directly.
   Integrity of data files are not verified, hence modified data files might introduce bugs into the program.
 
-[[Back to top](#index)]
+[[Back to top](#table-of-contents)]
 
+-------------------------------------
 
-<h1 id="faq">FAQ</h1>
+<h2 id="faq">FAQ</h2>
 ---------------------
 **Q:** On a scale of 1 to 10, how awesome is _Ez-Schedule_?  
 **A:** I would say off the charts, but it looks like 10 is the highest :v:
 
 **Q:** What is one example of someone who would use this?  
 **A:** A secretary who needs to keep track the boss's meetings,
-       and inform the boss of what the next meeting is for.
+and inform the boss of what the next meeting is for.
 
 **Q:** I am not allowed to add an event with no start/end time. Why?  
 **A:** Events with no start/end time are not the focus of _Ez-Schedule_,
-       since these events inherently cannot be scheduled.
+since these events inherently cannot be scheduled.
 
 **Q:** The `recur` feature only allow me to repeat in certain frequency. It does not have the frequency I want.  
 **A:** Oh no, that is sad to hear. As a extra feature designed for user convenience,
-       we cannot possibly support everyone's preferred frequency. 
-       In case that you really need to have events repeating at an unsupported frequency,
-       you can manually add the events one by one. (You type fast, right? :sunglasses:)
+we cannot possibly support everyone's preferred frequency.
+In case that you really need to have events repeating at an unsupported frequency,
+you can manually add the events one by one. (You type fast, right? :sunglasses:)
 
-**Q:** The calendar only displays a one or two events only, 
-       even though I have more events on that day. Is this a bug?  
-**A:** As with other conventional calendar applications (eg. Google/Outlook calendar), 
-       the UI has limited space and thus almost impossible to display all events on that day.
+**Q:** The calendar only displays a one or two events only,
+even though I have more events on that day. Is this a bug?  
+**A:** As with other conventional calendar applications (eg. Google/Outlook calendar),
+the UI has limited space and thus almost impossible to display all events on that day.
 
 **Q:** You say this is optimised for CLI use, but there is no command to transverse the calendar?  
-**A:** The calendar UI is a value-added feature and is not meant to totally replace the `find` feature, 
-       which allows the user to focus directly to a specific day.
+**A:** The calendar UI is a value-added feature and is not meant to totally replace the `find` feature,
+which allows the user to focus directly to a specific day.
```

Comment:
As mentioned earlier, the leading spaces are to increase readability (for us). Does not affect actual html.